client/components/dataviews/style.scss

```diff
@@ -205,3 +205,13 @@
 .dataviews-settings-section:has(.dataviews-settings-section__content:empty) {
 	display: none;
 }
+
```

Comment:
We should distinguish code that is a hotfix for an existing bug in dataviews vs custom code a dotcom page/section/component. With the current code organization, it's hard is to understand which is which, so I'd like to try the following: centralize hotfixes in the `components/dataviews`.

Each hotfix should have a corresponding Gutenberg issue (it'd great if it also comes with a PR). This way they are identifiable and can be removed when updating dataviews.

Reply:
@cleacos I'm unable to access the referrals screens. Would you push the changes for referrals here?

Reply:
I agree, all the core style fixes should be there: `components/dataviews/style.css`